source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -1379,17 +1397,20 @@ namespace AdaptiveCards { namespace Uwp
 
         ABI::AdaptiveCards::Uwp::ContainerStyle containerStyle;
         THROW_IF_FAILED(adaptiveColumn->get_Style(&containerStyle));
+        bool hasExplicitContainerStyle = true;
         if (containerStyle == ABI::AdaptiveCards::Uwp::ContainerStyle::None)
         {
+            hasExplicitContainerStyle = false;
             ABI::AdaptiveCards::Uwp::ContainerStyle parentContainerStyle;
             THROW_IF_FAILED(renderArgs->get_ContainerStyle(&parentContainerStyle));
             containerStyle = parentContainerStyle;
         }
         ComPtr<IAdaptiveRenderArgs> newRenderArgs;
         THROW_IF_FAILED(MakeAndInitialize<AdaptiveRenderArgs>(&newRenderArgs, containerStyle));
 
+        // If container style was explicitly assigned, apply background
         ABI::Windows::UI::Color backgroundColor;
-        if (SUCCEEDED(GetBackgroundColorFromStyle(containerStyle, m_hostConfig.Get(), &backgroundColor)))
+        if (hasExplicitContainerStyle && SUCCEEDED(GetBackgroundColorFromStyle(containerStyle, m_hostConfig.Get(), &backgroundColor)))
```

Comment:
>(hasExplicitContainerStyle && SUCCEEDED(GetBackgroundColorFromStyle(containerStyle, m_hostConfig.Get(), &backgroundColor))) [](start = 11, length = 123)

I like how this looks. Would you mind changing the one in BuildContainer to behave the same?